examples/integration_with_error.py

```diff
@@ -42,51 +44,38 @@
 data /= 0.5 * 0.25  # the dt in the integral for both dims
 data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
 # }}}
+data.ft("t2", shift=True)
+dt = data.get_ft_prop("t2","dt")
+# {{{ vector-normalize the FT
+data /= sqrt(ndshape(data)['t2']) * dt
+# }}}
 # {{{ First, run the code that automatically chooses integration bounds
 # and also assigns error
 fl.next("compare manual vs. automatic", legend=True)
-error_pathway = (
-    set(((j, k) for j in range(2) for k in range(4)))
-    - set(excluded_pathways)
-    - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
+error_pathway = (set(((j, k) for j in range(ndshape(data)['ph1']) for k in range(ndshape(data)['ph2'])))
+        - set(excluded_pathways)
+        - set([(signal_pathway['ph1'], signal_pathway['ph2'])])
 )
-error_pathway = [{"ph1": j, "ph2": k} for j, k in error_pathway]
-data = integral_w_errors(data, signal_pathway, error_pathway, fl=fl)
-fl.plot(data, ".", label="auto: real", capsize=6)
-fl.plot(data.imag, ".", label="auto: imaginary", capsize=6)
+error_pathway = [{'ph1':j, 'ph2':k} for j,k in error_pathway]
+s_int, returned_frq_slice = integral_w_errors(data, signal_pathway, error_pathway,
+        fl=fl, return_frq_slice=True)
+fl.plot(s_int, ".", label="fully auto: real", capsize=6)
+fl.plot(s_int.imag, ".", label="fully auto: imaginary", capsize=6)
```

Comment:
here I renamed some stuff and accounted for the fact that I tightened up the output of integral_w_error